user.py

```diff
@@ -0,0 +1,72 @@
+"""User information"""
+import sys
+from datetime import datetime
+import requests
+from lxml import html
+
+URL = 'https://www.airblue.com/'
+PATH = '//*[@id="content"]/span[2]/select[1]/option'
+
+
+def get_destinations(url, path, key='directions'):
+    """a method that is finding available destinations"""
+    try:
+        request = requests.get(url)
+    except ConnectionError as i:
+        print(i)
+        request = 'no response'
+        sys.exit()
+    body = html.fromstring(request.text)
+    destinations = {'directions': [], 'iata': []}
+    for i in range(2, 12):
+        destinations['directions'].append(''.join(body.xpath(f'{path}[{i}]/text()')))
+        destinations['iata'].append(''.join(body.xpath(f'{path}[{i}]/@value')))
+    return destinations[key]
+
+
+IATA_CODES = get_destinations(URL, PATH, 'iata')
+DESTINATIONS = get_destinations(URL, PATH)
+
+
+def user():
+    """a method that is getting information from user"""
+    print('hello! here are the destinations provided by', URL + ':')
+    print(' - ' + '\n - '.join(DESTINATIONS))
+    print('\nwhere would you like to go?')
+    while True:
+        iata_from = input('\nenter the IATA code of the initial airport: ').upper()
+        iata_to = input('enter the IATA code of the ultimate airport: ').upper()
+        try:
+            if iata_from not in IATA_CODES or iata_to not in IATA_CODES:
+                print('\nincorrect IATA! enter IATA codes again.')
+                continue
+            current_date = datetime.now().date()
+            latest_date = datetime.strptime('28-10-2020', '%d-%m-%Y').date()
```

Comment:
I've checked it several times and this date is still the last possible (the flights are not available starting with this date even today)

Reply:
we don't want to get into situation when there's no results just because hardcoded `latest_date` became the past.
Either use dynamic latest date(say, half-year ahead) or don't use it at all.